exercises/B-boolean-literals/exercise.js

```diff
@@ -6,7 +6,9 @@
 */
 
 let codeYourFutureIsGreat = true;
-
+let mozafarIsCool = false;
+let calculationCorrect = true;
+let moreThan10Students = true;
```

Comment:
I think as it write in the instruction, it should be false

Reply:
Hi Leila, thank you It's false, maybe this is the previous file not updated.